include/mbedtls/ssl.h

```diff
@@ -783,6 +783,22 @@ typedef int mbedtls_ssl_async_resume_t( mbedtls_ssl_context *ssl,
 typedef void mbedtls_ssl_async_cancel_t( mbedtls_ssl_context *ssl );
 #endif /* MBEDTLS_SSL_ASYNC_PRIVATE */
 
+#if defined(MBEDTLS_X509_CRT_PARSE_C)
```

Comment:
Ok, I'll add `MBEDTLS_SSL_KEEP_PEER_CERTIFICATE` to indicate the scope of these `#define`s. I'll also used  `MBEDTLS_KEY_EXCHANGE__WITH_CERT__ENABLED` instead of `MBEDTLS_X509_CRT_PARSE_C`.

Reply:
Done.